src/loki_alert_rules/hacluster_no_quorum.rule

```diff
@@ -0,0 +1,10 @@
+groups:
+  - name: haproxy_no_quorum
+    rules:
+      - alert: HaproxyNoQuorumError
+        expr: rate({%%juju_topology%%} |~ `resource management disabled due to lack of quorum` [5m]) > 0
+        for: 0s
```

Comment:
Is this intentional for all alerts? Should they be raised as soon as the expr is true?